### DISCUSSION TITLE
an material density file made through ESTAR

### DISCUSSION THREAD

**Discussion by yoongschoi** [nrc-cnrc/EGSnrc#1027] [Questions]
I made a material using ESTAR and saved the files

- in .density in egs_home/pegs4/density, following https://github.com/nrc-cnrc/EGSnrc/wiki/Generating-density-effect-correction-files-with-ESTAR, and 
- in .pegs4dat in egs_home/pegs4/data (I did it as well because only pegs4dat file can be open in PEGS file in egs_gui).

However, the material is not recognised when running the code. How can it be read properly?

plus, the output file from ESTAR doesn't look like the example file shown in the link above (Generating density effect correction files with ESTAR). I didn't change the texts since I thought the valid part to be read in the density file will be the data part, after the three lines of text. The data file is written like this:

ESTAR: Stopping Powers and Range Tables for Electrons

MGS-1 (Density=1.29 g/cm3) (I=154.9 eV)

Kinetic Energy (MeV)    D. Effect Parameter

 1.000E-02 0.000E+00 
 1.250E-02 0.000E+00 
.
.
.

Should I fix the three lines of the text like the example, or can I just leave it this way?

**Comment by ftessier** [comment]
Yes, you have to adjust the three lines as per the wiki instructions you cited:

> 7. This data is the foundation of your new density correction file for EGSnrc. **However, it is necessary to insert the 3-line file header expected by EGSnrc:**
